reporthandling/results/v1/reportsummary/controlsummarymethods.go

```diff
@@ -11,20 +11,26 @@ import (
 
 // GetStatus get the control status. returns an apis.ScanningStatus object
 func (controlSummary *ControlSummary) GetStatus() apis.IStatus {
-	if controlSummary.Status == apis.StatusUnknown {
+	if controlSummary.StatusInfo == (apis.StatusInfo{}) || controlSummary.StatusInfo.InnerInfo == "" {
```

Comment:
1. Remove the first condition (`controlSummary.StatusInfo == (apis.StatusInfo{}`))
2. Check if `controlSummary.Status != UnkwonStatus` ->
          `controlSummary.StatusInfo.Status = controlSummary.Status`